src/widget/wcoverartmenu.cpp

```diff
@@ -82,6 +81,29 @@ void WCoverArtMenu::slotChange() {
     coverInfo.source = CoverInfo::USER_SELECTED;
     coverInfo.coverLocation = selectedCoverPath;
     coverInfo.setImage(image);
+
+    QMessageBox::StandardButton saveButton = QMessageBox::question(nullptr,
```

Comment:
I think you can add a check if the cover is not already in the location before asking. 
If the picked cover is in a different location, we need to consider if it is OK to overwrite the existing. 

This is similar to change track metadata on one hand, on the other hand, we can't be sure if the file is not referenced from something else.

Reply:
Fair Enough. Normally with my tests the cover art overwritten again and again, so it is a must to have these checks. 

If they are in the same location no need to ask about it.
If the cover art has the same name we should ask if is it okay to overwrite it - same pop up with the different message maybe?

Is there a way for us to check if the file is referenced?